### PR TITLE
Add Router functionality

### LINKED ISSUE #12: Connect components with router and buttons with URL paths

### PR DESCRIPTION
**Pull request by BriBourassa**
** Change summary: ** 
- created routes and paths to navigate through a day's story

Closes #12

### CODE REVIEW

**Review by jsahim — APPROVED** [review]
Looks good!

**Comment by Jesusborjas006** [comment]
Setting up router looks good!

**Review by Jesusborjas006 — APPROVED** [review]

File: src/components/Choices/Choices.js [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
-      <button>Consult The Cat Bible</button>
+      <NavLink to="/result">
+        <button>Good Option Text</button>
+      </NavLink>
+      <NavLink to="/result">
+        <button>Neutral Option Text</button>
+      </NavLink>
+      <NavLink to="/result">
+        <button>Evil Option Text</button>
+      </NavLink>
```
Comment (qrispi) [inline src/components/Choices/Choices.js:17]: Love me some NavLinks

File: src/components/Story/Story.js [hunk 2]
```diff
@@ -1,17 +1,34 @@
...
+    // </div>
+    <>
+      <Switch>
+        <Route exact path="/">
+          <Day />
+        </Route>
+        <Route path="/choices">
+          <Choices />
+        </Route>
+        <Route path="/result">
```
Comment (qrispi) [inline src/components/Story/Story.js:26]: We'll need to interpolate the day # later but I love how this is working now!

**Review by qrispi — APPROVED** [review]
This is really coming together!

**Pull request merged**